src/commands/ip_allow.ts

```diff
@@ -0,0 +1,203 @@
+import yargs from 'yargs';
+import { CommonProps, ProjectScopeProps } from '../types';
+import { writer } from '../writer.js';
+import { Project, ProjectUpdateRequest } from '@neondatabase/api-client';
+import { projectUpdateRequest } from '../parameters.gen.js';
+import { log } from '../log.js';
+
+const IP_ALLOW_FIELDS = [
+  'id',
+  'name',
+  'IP_addresses',
+  'primary_branch_only',
+] as const;
+
+export const command = 'ip-allow';
+export const describe = 'Manage IP Allow';
+export const builder = (argv: yargs.Argv) => {
+  return argv
+    .usage('$0 ip-allow <sub-command> [options]')
+    .options({
+      'project-id': {
```

Comment:
@duskpoet @lpetkov Earlier I suggested `projectId` as positional argument [here](https://github.com/neondatabase/cloud/issues/8882#issuecomment-1891930122), but looking at other commands, (other than `projects`) we pass the `projectId` as option. Example - [Branch List](https://neon.tech/docs/reference/cli-branches#list) and [Connection String](https://neon.tech/docs/reference/cli-connection-string#the-connection-string-command)

Keeping it consistent with other commands as Ed suggested.

Reply:
Hey, I noticed that you have "--projectId", but elsewhere we use "--project-id":

neonctl branches list --project-id square-cell-51292327